elements/pfe-band/src/pfe-band.scss

```diff
@@ -303,8 +303,7 @@ $regions-grid: (
 }
 
 .pfe-band__aside {
-  flex-grow: 1;
-  align-self: flex-start;
+  height: 100%;
```

Comment:
Was this supposed to make it into this PR? I didn't ask when we were chatting, but any reason we didn't use `align-items: stretch;`?

Reply:
The container doesn't use flexbox at all so that's why I switched it for `height: 100%`.  This is what enables the jump links to work in the aside region of the band.

Reply:
Gotcha 👍 
Sounds good to me.